### PR TITLE
💄 ✏ 🖼 ML Logo Instead of Child Mind Institute Logo for Mindlogger Icon

### PR DESCRIPTION
**Pull request by shnizzedy**
- ML Logo Icon for iOS and Android
- Synchronize app version at 1.0 for iOS and Android

### CODE REVIEW

File: app/containers/activity/act.js [hunk 1]
```diff
@@ -102,13 +103,14 @@ class Act extends Component {
       "devices:os":`devices:${DeviceInfo.getSystemName()}`,
       "devices:osversion":DeviceInfo.getSystemVersion(),
       "deviceModel":DeviceInfo.getModel(),
+      "appVersion": packageJson.name + packageJson.version,
```
Comment (erik-ilyin) [inline app/containers/activity/act.js]: This is not correct app version. by doing so, we will have 3 different app versions. - package.json, ios, android

Comment (shnizzedy) [reply app/containers/activity/act.js]: @erik-ilyin can we just import the package.json version in iOS and Android to keep them in sync?

Comment (shnizzedy) [reply app/containers/activity/act.js]: Or do you have a suggested solution to just have one version number?

Comment (erik-ilyin) [reply app/containers/activity/act.js]: no we can't import version in iOS and android. we should set version separate in iOS and android. However we can put same version name for display in IOS and Android and then import that version name here. Please remove these part for now. I will implement that later.

Comment (shnizzedy) [reply app/containers/activity/act.js]: > Please remove these part for now.

[Done](https://github.com/ChildMindInstitute/mindlogger-app/pull/51/commits/4a21abe2349ad22eb602596cc9bfb4f80cd7d042).

File: package.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
   "hash-int": "^1.0.0",
   "instabug-reactnative": "^8.0.19",
   "moment": "^2.13.0",
-  "native-base": "^2.6.0",
+  "native-base": "^2.4.0",
```
Comment (erik-ilyin) [inline package.json]: why do we change this native-base version?

Comment (shnizzedy) [reply package.json]: 2.6 turned the hamburger and reload icons to question marks on iOS

Comment (shnizzedy) [reply package.json]: 5398abb sets native-base back to ^2.6.0

**Pull request merged**